documentation/api.md

```diff
@@ -76,3 +76,15 @@ CRUD for Visual Rules. Originally detailed in [DCA documentation](https://colabo
 | ------ | :----------------------- | ----------------------------------- |
 | PUT    | /admin/log?level={level} | Set log level of perseo             |
 | GET    | /admin/log               | Get the current log level of perseo |
+
+### Pagination
+
+Perseo-fe implements a pagination mechanism in order to help clients to retrieve large sets of resources. This mechanism
+works for all listing operations in the API (GET /rules, GET /vrules)
+
+The mechanism is based on three URI parameters:
+
+-   **limit**: in order to specify the maximum number of elements (default is 20, maximum allowed is 1000).
+-   **offset**: in order to skip a given number of elements at the beginning (default is 0)
+-   **count** (as option): if activated then a Fiware-Total-Count header is added to the response, with a count of total
```

Comment:
Is count really used?

Looking into the PR I only see req.query.limit and req.query.offset but not req.query.options, although maybe it was implemented in a past PR.

Reply:
I'm going to open a new PR to include options=count since already count is always wired to true.

Reply:
fixed in https://github.com/telefonicaid/perseo-fe/pull/396/commits/14b89e5c4f35337872f48f2f8b7000071e968eaf